CONTRIBUTING.md

```diff
@@ -1,5 +1,18 @@
 # Contributing to cellmlmanip
 
+##Developer installation
+Users install chaste_codegen using pip. For instructions on installing as a user, see README.md
+
+For developers, to set up a virtual environment and install requirements:
+
+conda create -n cellmlmanip python=3.6
```

Comment:
I don't think we should include virtual environment info here, especially not with conda (which is not a requirement for cellmlmanip)